doc/api/perf_hooks.md

```diff
@@ -162,6 +172,40 @@ added: REPLACEME
 The [`timeOrigin`][] specifies the high resolution millisecond timestamp from
 which all performance metric durations are measured.
 
+### performance.timerify(fn)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `fn` {Function}
+
+Wraps a function within a new function that measures the running time of the
+wrapped function. A `PerformanceObserver` must be subscribed to the `'function'`
+event type in order for the timing details to be accessed.
+
+```js
+const {
+  performance,
+  PerformanceObserver
+} = require('perf_hooks');
+
+function someFunction() {
+  console.log('hello world');
+}
+
+const wrapped = performance.timerify(someFunction);
+
+const obs = new PerformanceObserver((list) => {
+  console.log(list.getEntries()[0].duration);
+  obs.disconnect();
+  performance.clearFunctions();
```

Comment:
Need not this call also be added in the `Class: PerformanceObserver(callback)` example below (lines 433-436)?